README.md

```diff
@@ -60,6 +60,13 @@ Calls `aws cloudformation validate-template`
 
 Controls what cloudformation action to apply on the stack. This config is a required parameter. 
 
+### skip-deploy
+* **BitOps Property:** `skip-deploy`
+* **Environment Variable:** `CFN_SKIP_DEPLOY`
+* **default:** `none`
+* **Required:** `false`
```

Comment:
I see that other config options in the README mention `Required` only when it's `true` which makes sense to me to avoid overloading the docs.

Let's follow the same consistent way for `skip-deploy` too:
```suggestion
```